frontend/packages/console-app/src/__tests__/network-policies/OWNERS

```diff
@@ -0,0 +1,10 @@
+reviewers:
+  - jotak
+  - mariomac
+  - OlivierCazade
+  - spadgett
+approvers:
+  - jotak
+  - spadgett
```

Comment:
You can remove me as an approver since I'm an approver in the root OWNERS file for the repo.